2-Webpack/src/index.js

```diff
@@ -0,0 +1,190 @@
+const storage = require('./modules/storage.js')
+const {getValue,setValue,getInnerText,setInnerText} = require('./modules/helper-functions.js')
+
+let editedID = 0;
+let editMode = false;
+
+function getNewID(){
+    return new Date().getTime().toString()
+}
+function addItem(){
+    const title = getValue('title-input')
+    if (title == ''){
+        alert('cant add an item without a title')
+        return
+    }
+    if (editMode){
+        alert('cant add an item while in edit mode')
+        return
+    }
+    const content = getValue('content-input')
+    const list = document.getElementById('list')
+    const id = getNewID()
+    storage.set(id, title,content)
+    const itemElement = createTodoItemElement(id,title,content)
+    list.appendChild(itemElement)
+    // reset menu display
+    clearMenu()
+    setInnerText('apply-button','Add')
+}
+function deleteCheckedItems(){
+    const itemList = storage.getAll()
+    const list = document.getElementById('list')
+    for (const id in itemList) {
+        const checkbox = document.getElementById(`item-checkbox${id}`)
+        if(checkbox.checked){
+            console.log(id, 'is checked')
+            const itemElement = document.getElementById(`${id}`)
+            list.removeChild(itemElement)
+            storage.remove(id)
+        }
+    }
+}
+function cancelEdit(){
+    clearMenu()
+    editMode = false
+    setInnerText('apply-button','Add')
+    document.getElementById('cancel-button').hidden = true
+    editedID = 0  
+}
+
+function clearMenu(){
+    setValue('title-input','')
+    setValue('content-input','')
+}
+
+function menuButtonClick(){
+    if(!editMode){
+        // adding a new item
+        addItem()
+    }else{
+        // editing an existing item
+        if (getValue('title-input')==''){
```

Comment:
double equal doesn't compare types and may lead to unexpected result, for example:
`false == '' // true`
`[] == '' // true`